### DISCUSSION TITLE
v2.2.1

### DISCUSSION THREAD

**Discussion by KRTirtho** [KRTirtho/spotube#108] [Announcements]
### Improved 📈
- Page transitions defaulted to material you design

### Bug fixes 🐛
- Mini Player flickering on random state updates
- Track More Options not showing when not logged in
- Wrong link to Client ID & Client Secret tutorial in Login page
- Changing preferences in Settings resets the entire Playback

**Full Changelog**: https://github.com/KRTirtho/spotube/compare/v2.2.0...v2.2.1

<hr /><em>This discussion was created from the release <a href='https://github.com/KRTirtho/spotube/releases/tag/v2.2.1'>v2.2.1</a>.</em>

**Comment by Xsodu** [comment]
Mini Player keeps flickering on Android

**Comment by KRTirtho** [comment]
@Xsodu It's a known issue & I'm currently finding ways to mitigate this issue
